test/test_prototype_transforms.py

```diff
@@ -286,35 +286,6 @@ def test__transform_culling(self, mocker):
         assert_equal(output["masks"], masks[is_valid])
         assert_equal(output["labels"], labels[is_valid])
 
-    def test__transform_bounding_boxes_clamping(self, mocker):
```

Comment:
I've removed this test which was failing with 

```
    raise ValueError(f"Expected a 1D or 2D tensor, got {tensor.ndim}D")
ValueError: Expected a 1D or 2D tensor, got <MagicMock name='clamp_bounding_boxes().ndim' id='140248661013024'>D
```


I couldn't find out why - all the tensors that were passed there were all 2D. I decided to remove the tests.

Reply:
Fixed it and put it back in 0189a6bb8ded08b86d74f6d09c974f2ec23f09a2